src/amr/data/particles/refine/split.h

```diff
@@ -12,6 +12,82 @@
 #include "core/data/particles/particle.h"
 #include "core/utilities/box/box.h"
 
+namespace PHARE::amr
+{
+enum class Color : uint8_t { BLUE, GREEN, RED, ORANGE, YELLOW, PINK, PURPLE, MAGENTA, BLACK, LIME };
```

Comment:
We should only keep the 6 patterns are described in the wiki : `BLACK`, `PINK`, `PURPLE`, `LIME`, `BROWN` and `WHITE`

Reply:
done